### PR TITLE
Use new api `CryptoApi.getCrossSigningStatus` in `CrossSigningPanel`

### PR DESCRIPTION
**Pull request by florianduros**
## Checklist

-   [x] Tests written for new code (and old code if feasible)
-   [x] Linter and other CI checks pass
-   [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

Needs https://github.com/matrix-org/matrix-js-sdk/pull/3452

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: test/test-utils/client.ts [hunk 1]
```diff
@@ -161,4 +161,15 @@ export const mockClientMethodsCrypto = (): Partial<
             isStoredInSecretStorage: jest.fn(),
         },
     },
+    getCrypto: jest.fn().mockReturnValue({
```
Comment (richvdh) [inline test/test-utils/client.ts:164]: can we remove `crossSigningInfo` above? or not yet?

File: test/components/views/settings/CrossSigningPanel-test.tsx [hunk 2]
```diff
@@ -38,7 +38,15 @@ describe("<CrossSigningPanel />", () => {
...
-        mocked(mockClient.crypto!.crossSigningInfo).isStoredInSecretStorage.mockClear().mockResolvedValue(null);
+        mocked(mockClient.getCrypto()!.getCrossSigningStatus).mockResolvedValue({
+            publicKeysOnDevice: true,
+            privateKeysInSecretStorage: false,
+            privateKeysCachedLocally: {
+                masterKey: true,
+                selfSigningKey: true,
+                userSigningKey: true,
+            },
+        });
```
Comment (richvdh) [inline test/components/views/settings/CrossSigningPanel-test.tsx:49]: isn't this the default, from `mockClientMethodsCrypto` ?

Comment (florianduros) [reply test/components/views/settings/CrossSigningPanel-test.tsx:49]: Yes but I'm overriding it in some tests. So in order to not disturb the other tests, I'm setting the default value here before each test

Comment (richvdh) [reply test/components/views/settings/CrossSigningPanel-test.tsx:49]: bleh; I find this quite confusing.

Can we move the setup of `mockClient` into `beforeEach`?

Comment (florianduros) [reply test/components/views/settings/CrossSigningPanel-test.tsx]: Yes done :)

**Review by richvdh — APPROVED** [review]
lgtm!

**Pull request merged**